src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/AvoidDeadConditionalCode.cs

```diff
@@ -72,13 +70,20 @@ public override void Initialize(AnalysisContext context)
 
                         if (operationRoot.HasAnyOperationDescendant(ShouldAnalyze))
                         {
-                            var cfg = operationRoot.GetEnclosingControlFlowGraph();
+                            // Skip duplicate analysis from operation blocks for constructor initializer and body.
```

Comment:
This is unfortunate side effect of the fact that we have different OperationBlocks for constructor initializer and constructor method body, but both of them are part of the same control flow graph. However, most analyzer authors should not require such duplicate checking, as unlike this analyzer which performs flow analysis on every operation block, most analyzers will do so only on specific operations.